packages/react/src/app-shell/AppShell.css.ts

```diff
@@ -0,0 +1,13 @@
+import { recipe, style } from "../vanilla-extract";
+
+export const appShell = recipe({
+  base: [
+    {
+      gap: "0",
+      h: "full",
+    },
+    style({
+      width: "100vh",
```

Comment:
Can't we use `w="full"` for this? or other way the story should handle the width.